Doc/library/tkinter.rst

```diff
@@ -188,6 +188,568 @@ the modern themed widget set and API::
       child widget names as the keys and the child instance objects as the
       values.
 
+   Methods inheritted from :class:`Misc` parent class (some methods ommited):
+
+   .. method:: destroy()
+
+      Destroy this and all descendants widgets. This will
+      end the application of this Tcl interpreter.
+
+   .. method:: after(ms, func=None, *args)
+
+      Call function once after given time.
+
+      MS specifies the time in milliseconds. FUNC gives the
+      function which shall be called. Additional parameters
+      are given as parameters to the function call.  Return
+      identifier to cancel scheduling with :meth:`~Tk.after_cancel`.
+
+   .. method:: after_idle(func, *args)
+
+      Call FUNC once if the Tcl main loop has no event to
+      process.
+
+      Return an identifier to cancel the scheduling with
+      :meth:`~Tk.after_cancel`.
+
+   .. method:: after_cancel(id)
+
+      Cancel scheduling of function identified with ID.
+
+      Identifier returned by :meth:`~Tk.after` or :meth:`~Tk.after_idle` must be
+      given as first parameter.
+
+   .. method:: bell(displayof=0)
+
+      Ring a display's bell.
+
+   .. method:: clipboard_get(**kw)
+
+      Retrieve data from the clipboard on window's display.
+
+      The window keyword defaults to the root window of the Tkinter
+      application.
+
+      The type keyword specifies the form in which the data is
```

Comment:
Note: at least on Windows, there is only one clipboard, even when there are multiple screens.
```suggestion
      Return data from the clipboard.

      The *window* keyword defaults to the root window of the Tkinter
      application.  The *type* keyword specifies the form in which the data is
```